src/main/java/com/Acrobot/ChestShop/Commands/AccessToggle.java

```diff
@@ -0,0 +1,40 @@
+package com.Acrobot.ChestShop.Commands;
+
+import com.Acrobot.ChestShop.Listeners.Player.PlayerInteract;
+import com.Acrobot.ChestShop.Permission;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+/**
+ * @author g--o
+ *
+ * Command for toggling chest access
+ */
+public class AccessToggle implements CommandExecutor {
+    public boolean onCommand(CommandSender sender, Command cmd, String label, String[] args) {
+        if (cmd.getName().equals("csaccess")) {
+            String resultMsg = "Access to shops is ";
+            Player player = (Player) sender;
+            String permNode = Permission.OTHER_NAME_ACCESS + ".*";
+
+            if (PlayerInteract.canOpenOtherShops(player)) { // @FIX: why is this deprecated?
```

Comment:
> why is this deprecated?

See the comment on the method: You should be using the actual utility method mentioned there directly. It was deprecated as all usage of that method was removed from the plugin in b853afbd5223d9f6885601c60e5ec438e6721926.

Reply:
well but now i use it lmao, it has no logical replacement